package.json

```diff
@@ -1,49 +1,77 @@
 {
-  "name": "@front-finance/front-react-native",
-  "version": "1.0.0",
-  "private": true,
-  "description": "Front Finance react native resources (SDK and example app)",
-  "workspaces": [
-    "./example",
-    "packages/*"
-  ],
+  "version": "1.2.1",
+  "name": "@front-finance/frontfinance-rn-sdk",
+  "description": "Front Finance React Native SDK.",
+  "private": false,
+  "license": "MIT",
+  "author": "Front Finance, Inc",
+  "repository": "https://github.com/FrontFin/front-b2b-link-rn.git",
+  "homepage": "https://github.com/FrontFin/front-b2b-link-rn",
+  "main": "index.js",
+  "typings": "index.d.ts",
+  "module": "index.js",
   "scripts": {
-    "typecheck": "lerna run typecheck",
-    "lint": "lerna run lint",
-    "version": "lerna version --no-private",
-    "build": "lerna run build",
-    "test": "lerna run test",
-    "start:ex": "cd example && yarn expo start"
+    "build": "tsc --build && node scripts/build.js",
+    "check-build": "node scripts/verify-build.js",
+    "clean": "tsc --build --clean",
+    "lint": "eslint --ext .js,.ts,.tsx ./src",
+    "test": "jest --passWithNoTests",
+    "types-check": "tsc --noEmit",
+    "publish:npm": "yarn build && cd dist && yarn publish --access public"
   },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/FrontFin/front-b2b-link-rn.git"
+  "husky": {
+    "hooks": {
+      "pre-commit": "yarn lint"
+    }
+  },
+  "prettier": {
+    "printWidth": 80,
+    "semi": true,
+    "singleQuote": true,
+    "trailingComma": "es5"
   },
   "keywords": [
     "front-finance"
   ],
-  "author": "Front Finance, Inc",
-  "license": "MIT",
-  "bugs": {
-    "url": "https://github.com/FrontFin/front-b2b-link-rn/issues"
+  "publishConfig": {
+    "access": "public"
+  },
+  "files": [
+    "dist"
+  ],
+  "engines": {
+    "node": ">=16"
+  },
+  "packageManager": "yarn@1.22.17",
+  "dependencies": {
+    "react-native-webview": "^11.26.0"
   },
-  "homepage": "https://github.com/FrontFin/front-b2b-link-rn#readme",
   "peerDependencies": {
-    "react": "*",
-    "react-native": "*"
+    "react": ">=16",
+    "react-native": ">=0.60"
   },
   "devDependencies": {
-    "@babel/cli": "^7.17.6",
-    "@babel/core": "^7.17.5",
-    "@typescript-eslint/eslint-plugin": "^6.1.0",
-    "@typescript-eslint/parser": "^6.1.0",
-    "eslint": "^8.45.0",
-    "eslint-config-prettier": "^8.8.0",
-    "eslint-plugin-import": "^2.27.5",
-    "eslint-plugin-prettier": "^5.0.0",
-    "lerna": "^6.6.2",
-    "metro-react-native-babel-preset": "^0.68.0",
-    "prettier": "^3.0.0",
+    "@babel/core": "^7.20.0",
+    "@babel/preset-env": "^7.20.0",
+    "@babel/runtime": "^7.20.0",
+    "@react-native/eslint-config": "^0.72.2",
+    "@react-native/metro-config": "^0.72.11",
+    "@testing-library/react-native": "^12.2.2",
+    "@tsconfig/react-native": "^3.0.0",
+    "@types/node": "^20.5.1",
+    "@types/react": "^18.0.24",
+    "@types/react-test-renderer": "^18.0.0",
```

Comment:
Plz add `“@types/jest”: “^29.5.4"` to devDependencies to resolve issue with jest